cmd/login.go

```diff
@@ -13,36 +13,47 @@ import (
 	"github.com/manifoldco/manifold-cli/session"
 )
 
+var errLoggedInEnv = cli.NewExitError("", 0)
+
 func init() {
 	loginCmd := cli.Command{
 		Name:     "login",
 		Usage:    "Log in to your account",
 		Category: "ADMINISTRATIVE",
-		Action:   login,
+		Subcommands: []cli.Command{
```

Comment:
Since we put the oauth commands under `oauth`, are these needed?
It's a bit weird to have it split.

Reply:
Also to note: It's going to be confusing to users when they run `manifold login` (as it will perform the login action) but `manifold [any-other-command]` would display the subcommands for that group.

So having `manifold login github` makes the UX unpredictable.